vignettes/VectorControl_Housing.Rmd

```diff
@@ -0,0 +1,125 @@
+       type = "l", col = cols[3], lwd = 1,
+       xlab = "Time (days)", ylab = expression(paste(italic(Pf),"PR"[2-10])),
+       xaxs = "i", yaxs = "i", ylim = c(0, 1))
+  lines(x = output_control$timestep, y = output_control$n_detect_730_3650 / output_control$n_730_3650,
+        col = cols[5], lwd = 1)
+  lines(x = output_nets_housing$timestep, y = output_nets_housing$n_detect_730_3650 / output_nets_housing$n_730_3650,
+        col = cols[1], lwd = 1)
+  abline(v = bednetstimesteps, col = "black", lty = 2, lwd = 1)
+  abline(v = housingtimesteps, col = "darkred", lty = 2, lwd = 1)
+  
+  text(x = bednetstimesteps + 10, y = 0.95, labels = "Bed net int.", adj = 0, cex = 0.8)
+  text(x = housingtimesteps + 10, y = 0.95, labels = "Housing int.", adj = 0, cex = 0.8)
+  grid(lty = 2, col = "grey80", lwd = 0.5)
+  legend("bottomleft", box.lty = 0, bg = "white",
+         legend = c("Prevalence for housing scenario",
+                    "Prevalence for nets and housing scenario",
+                    "Prevalence for control scenario"),
+         col = c(cols[3], cols[4],cols[1],cols[5]), lty = c(1,1,1,1), lwd = 2, cex = 0.8, y.intersp = 1.3)
+}
+```
+  
+## Setting housing parameters
+
+### Parameterisation
+
+Use the `get_parameters()` function to generate the list of parameters for a perennial profile, accepting the default values to run the simulation from an equilibrium starting point. 
+
+```{r}
+year <- 365
+sim_length <- 6 * year
+human_population <- 1000
+starting_EIR <- 50
+
+simparams <- get_parameters(
+  list(human_population = human_population)
+)
+
+simparams <- set_equilibrium(parameters = simparams, init_EIR = starting_EIR)
+
+output_control <- run_simulation(timesteps = sim_length, parameters = simparams)
+```
+
+#### A note on mosquito species  
+It is also possible to use the `set_species()` function to account for different mosquito species in the simulation. In this case, the matrices would need to have additional columns corresponding to each mosquito species. If you are not already familiar with the `set_species()` function, see the [Mosquito Species](https://mrc-ide.github.io/malariasimulation/articles/SetSpecies.html) vignette.  
+
+The default parameters are set to model *Anopheles gambiae*. 
+```{r}
+simparams$species
+simparams$species_proportions
+```
+  
+### Simulation
+
+Having established a base set of parameters, we can now create a copy of this parameter list and update it to specify that housing has been altered in some way to reduce access to mosquitoes. In the example below, we simulate that 30% of housing is 'improved' in the 2nd year, and that nets are used by a random 50% of the population every three years. We can change the 'repellence' introduced from households multiple times and differently for each mosquito species potentially.
+  
+```{r, fig.align = 'center', out.width='100%'}
+housingparams <- set_housing(
+  simparams,
+  timesteps = housingtimesteps,
```

Comment:
This needs to be defined somewhere.

You can check that all the automated checks will pass with `devtools::check`